packages/typescript-estree/src/useProgramFromProjectService.ts

```diff
@@ -34,6 +46,6 @@ export function useProgramFromProjectService(
   function absolutify(filePath: string): string {
     return path.isAbsolute(filePath)
       ? filePath
-      : path.join(projectService.host.getCurrentDirectory(), filePath);
+      : path.join(service.host.getCurrentDirectory(), filePath);
   }
```

Comment:
i realise this is late - but maybe `ensureAbsolutePath` would be better here, eg `ensureAbsolutePath(filePath, service.host.getCurrentDirectory())` just so we're reusing things.